src/main/java/com/alkemy/ong/controller/UserController.java

```diff
@@ -30,103 +25,88 @@
 import org.springframework.web.bind.annotation.RequestHeader;
 import org.springframework.web.bind.annotation.RequestPart;
 import org.springframework.web.bind.annotation.ResponseStatus;
-import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.multipart.MultipartFile;
-import static com.alkemy.ong.mapper.UserMapper.mapDomainToDTO;
-import static com.alkemy.ong.mapper.UserMapper.mapUpdateDTOToDomain;
-
-@RestController
-public class UserController {
-
-    private final UserService userService;
-
-    public UserController(UserService userService) {
-        this.userService = userService;
-    }
 
+@Tag(name = "Users", description = "Register, login and auth me Users")
```

Comment:
En description podemos poner simplemente una frase general, como `Operations related to Users` y que luego en el detalle de cada operación se vea qué hace cada una 👍🏼